### ISSUE TITLE
Uncaught #<OperationStackError> when trying to reapply last ciw or caw operation

### ISSUE THREAD

**Issue by gnl** [t9md/atom-vim-mode-plus#86]
1. Execute `ciw` or `caw` on a word, enter some text and ctrl+c to return to normal mode.
2. Go to another word and hit `.` to repeat last operation.
3. Operation is applied correctly, but:
4. Move down and see that visual mode has been entered.
5. Hit ctrl+c to exit visual mode, works fine.
6. Go to another word and hit `.` again to repeat last operation -> visual mode is entered again and exception is thrown.

All of this works as expected with the `w` text object, the error only occurs on `iw` and `aw`

**Atom Version**: 1.3.2
**System**: Mac OS X 10.10.5
**Thrown From**: [vim-mode-plus](https://github.com/t9md/atom-vim-mode-plus) package, v0.10.2

### Stack Trace

Uncaught #<OperationStackError>

```
At /Users/gnl/.atom/packages/vim-mode-plus/lib/operation-stack.coffee:42

undefined
```

### Commands

```
     -0:14.6.0 vim-mode-plus:move-to-previous-word (atom-text-editor.editor.vim-mode-plus.is-focused.normal-mode)
     -0:14.2.0 vim-mode-plus:repeat (atom-text-editor.editor.vim-mode-plus.is-focused.normal-mode)
  2x -0:13.7.0 vim-mode-plus:move-down (atom-text-editor.editor.vim-mode-plus.is-focused.visual-mode.characterwise)
     -0:13.1.0 vim-mode-plus:activate-normal-mode (atom-text-editor.editor.vim-mode-plus.is-focused.visual-mode.characterwise)
  2x -0:12.8.0 vim-mode-plus:move-up (atom-text-editor.editor.vim-mode-plus.is-focused.normal-mode)
     -0:12 vim-mode-plus:move-down (atom-text-editor.editor.vim-mode-plus.is-focused.normal-mode)
     -0:11.5.0 vim-mode-plus:repeat (atom-text-editor.editor.vim-mode-plus.is-focused.normal-mode)
     -0:10.8.0 vim-mode-plus:reset-normal-mode (atom-text-editor.editor.vim-mode-plus.is-focused.normal-mode)
  7x -0:08.3.0 vim-mode-plus:move-up (atom-text-editor.editor.vim-mode-plus.is-focused.normal-mode)
  2x -0:07.1.0 vim-mode-plus:move-down (atom-text-editor.editor.vim-mode-plus.is-focused.normal-mode)
  2x -0:06.0 vim-mode-plus:move-right (atom-text-editor.editor.vim-mode-plus.is-focused.normal-mode)
  2x -0:05.5.0 vim-mode-plus:undo (atom-text-editor.editor.vim-mode-plus.is-focused.normal-mode)
     -0:04.1.0 vim-mode-plus:reset-normal-mode (atom-text-editor.editor.vim-mode-plus.is-focused.normal-mode)
  2x -0:04.0 vim-mode-plus:move-right (atom-text-editor.editor.vim-mode-plus.is-focused.normal-mode)
     -0:02.7.0 vim-mode-plus:change (atom-text-editor.editor.vim-mode-plus.is-focused.normal-mode)
     -0:02.6.0 vim-mode-plus:a-word (atom-text-editor.editor.vim-mode-plus.is-focused.operator-pending-mode)
```

### Config

```json
{
  "core": {
    "openEmptyEditorOnStart": false,
    "projectHome": "/Users/gnl/Documents",
    "disabledPackages": [
      "ide-purescript",
      "language-purescript",
      "language-java",
      "language-go",
      "language-c",
      "language-clojure",
      "language-csharp",
      "language-mustache",
      "language-make",
      "language-perl",
      "language-python",
      "language-ruby",
      "language-ruby-on-rails",
      "language-toml",
      "language-yaml",
      "wrap-guide",
      "tabs",
      "background-tips",
      "welcome",
      "open-on-github",
      "floobits",
      "color-picker",
      "metrics",
      "package-generator",
      "styleguide",
      "linter-elm-make",
      "language-elm",
      "ide-haskell",
      "ide-haskell-repl",
      "ide-haskell-cabal",
      "haskell-hoogle",
      "haskell-ghc-mod",
      "language-haskell",
      "autocomplete-haskell",
      "linter-hlint",
      "haskell-pointfree",
      "spell-check",
      "update-package-dependencies",
      "snippets",
      "asciidoc-preview",
      "hover-tooltips-hdevtools",
      "linter-hdevtools",
      "php-debug",
      "language-swift",
      "linter-swiftc",
      "swift-debugger"
    ],
    "packagesWithKeymapsDisabled": [
      "sensible-tab",
      "last-cursor-position",
      "goto-last-edit",
      "zentabs",
      "external-open",
      "linter-flow",
      "git-diff",
      "git-history",
      "tree-view-git-modified",
      "autocomplete-plus",
      "script",
      "term3",
      "open-on-bitbucket",
      "project-manager",
      "goto",
      "ide-flow",
      "tab-switcher",
      "symbols-view",
      "atom-beautify",
      "scratch",
      "lazy-motion",
      "project-ring",
      "blame",
      "linter",
      "merge-conflicts",
      "clang-format",
      "recent-files-fuzzy-finder",
      "git-plus"
    ],
    "themes": [
      "one-dark-ui",
      "slime"
    ]
  },
  "vim-mode-plus": {
    "useSmartcaseForSearch": true,
    "showHoverOnOperate": true,
    "showHoverSearchCounter": true,
    "numberRegex": "(?:\\B-)?[0-9] ",
    "stayOnTransformString": true,
    "flashScreenOnSearchHasNoMatch": false,
    "flashOnSearch": false,
    "showHoverSearchCounterDuration": 1000,
    "incrementalSearch": true,
    "flashOnUndoRedo": true,
    "stayOnYank": true
  }
}
```

### Installed Packages

```coffee
# User
Hydrogen, v0.6.5
atom-fuzzy-grep, v0.9.0
atom-ternjs, v0.12.1
clip-history, v0.1.12
cursor-blink-interval, v0.3.1
cursor-history, v0.5.9
external-open, v0.6.1
file-icons, v1.6.13
git-diff-details, v0.21.0
hyperclick, v0.0.35
ide-flow, v0.8.0
imdone-atom, v1.3.15
js-hyperclick, v1.4.1
language-asciidoc, v0.12.0
language-javascript-semantic, v0.2.1
lazy-motion, v0.1.16
linter, v1.11.3
linter-eslint, v5.2.6
linter-flow-plus, v3.1.0
linter-phpmd, v1.3.2
mac-fix-path, v0.0.0
open-on-bitbucket, v0.4.0
package-settings, v1.0.1
package-sync, v1.0.0
pigments, v0.19.5
react, v0.12.10
recent-files-fuzzy-finder, v0.2.4
remember-file-positions, v0.2.1
slime, v1.4.0
symbol-gen, v1.0.0
tab-switcher, v1.3.3
tablr, v0.9.0
tabs-to-spaces, v1.0.1
vim-mode-plus, v0.10.2
zentabs, v0.8.6

# Dev
No dev packages
```

**Comment by t9md** [comment]
Bug introduced when fixing SelectLatestChange operator.

https://github.com/t9md/atom-vim-mode-plus/commit/5933a4992fd5cf857e65314cc927b77f02f87231

**Comment by t9md** [comment]
included in v0.11.0

**Comment by gnl** [comment]
Damn you're quick...works great now, thanks!
